setup.py

```diff
@@ -117,11 +117,12 @@ def _path_under_setup(*args):
     url=url,
     license=license,
     packages=[pkg_name] + tests,
-    install_requires=['numpy'] + (['cython'] if USE_CYTHON else []),
-    setup_requires=['numpy'] + (['cython'] if USE_CYTHON else []),
-    extras_require={'docs': ['Sphinx', 'sphinx_rtd_theme', 'numpydoc']},
+    include_dirs=['levmar/levmar-2.6', np.get_include()],
     ext_modules=ext_modules,
-    zip_safe=False,
+    data_files=[('lib/site-packages/' + pkg_name, glob('libs/*.dll'))],
```

Comment:
this doesn't look particularily general?

Reply:
true, but this comes with the problem of the lack of availability of Win32/64 LAPACK binaries. see below, I will conda-forge give a go